docker-micro-benchmark/helpers/docker_helpers.go

```diff
@@ -0,0 +1,263 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"sync"
+	"time"
+
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/juju/ratelimit"
+)
+
+func newContainerName() string {
+	return "benchmark_container_" + strconv.FormatInt(time.Now().UnixNano(), 10) + strconv.Itoa(rand.Int())
+}
+
+// CreateContainers creates num of containers
+func CreateContainers(client *docker.Client, num int) []string {
+	ids := []string{}
+	for i := 0; i < num; i++ {
+		name := newContainerName()
+		dockerOpts := docker.CreateContainerOptions{
+			Name: name,
+			Config: &docker.Config{
+				AttachStderr: false,
+				AttachStdin:  false,
+				AttachStdout: false,
+				Tty:          true,
+				Cmd:          []string{"/bin/bash"},
+				Image:        "ubuntu",
```

Comment:
Actually, ubuntu is a pretty big image. Maybe we can use something more lightweight like alpine?